features/pantheon-logged-out.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Verify various Pantheon features as a logged-out user
+
+  Scenario: Cache-Control should default to TTL=600
+    When I go to "/"
+    Then the response header "Cache-Control" should exist
+    And the response header "Cache-Control" should contain "max-age=600"
```

Comment:
Not necessarily. I've made it more precise in d4009cc7d5913f83dd7050c18ba327f019ca937b